lib/engine/step/train.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 require_relative 'base'
+require_relative 'trainer'
 
 module Engine
   module Step
     class Train < Base
+      include Trainer
```

Comment:
lol trainer is not a great name, it is like saying this step likes to work out at the gym

maybe TrainBuyer

Reply:
i don't see why this was split out, isn't the only other use casse in the special track subclass?

Reply:
I thought token -> tokener
track -> tracker
train -> trainer....

But in the new commit I'll try step buy_train that uses a module train.

Now I felt I needed having the step + module pattern, which in part was due to the track / token patern, and in part that I did not get it to work subclassing Step::Train.

But if you still think it is a bad idea I can give it a new try.

In any case, isn't Step::BuyTrain a better name than Step::Train?